live-site/src/pages/index.tsx

```diff
@@ -55,14 +54,13 @@ const IndexPage: React.FC = () => {
       {isDesktop && new Date() > new Date('2023-02-10T17:00:00-05:00') && (
         <HackingRemaining />
       )}
-      <Background isDay={true} />
       <ComingUpSection />
       <WelcomeSection />
       <EventScheduleSection />
       <ResourcesSection />
       <MeetTheTeamSection />
       <MentorsSection />
-      <Footer tabs={liveSiteTabInfo} isDay />
+      <Footer tabs={liveSiteTabInfo} isDay={true} />
```

Comment:
if the prop is true we can just leave it as isDay I think??? 

it's the same thing as isDay={true}